shell/probe/raindrop.ps1

```diff
@@ -1,14 +1,24 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
 function Execute {
-    Param([String]$File)
+    Param([String]$file)
```

Comment:
https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.4#use-pascal-case-for-parameter-names

I don't make the rules 🤷